bloom/commands/git/import_upstream.py

```diff
@@ -205,6 +206,12 @@ def handle_tree(tree, directory, root_path, version):
                 file_data = show(BLOOM_CONFIG_BRANCH, os.path.join(root_path, rel_path))
             # Write file
             with open(rel_path, 'wb') as f:
+                # Python 2 will treat this as an ascii string but
+                # Python 3 will not re-decode a utf-8 string.
+                if sys.version_info.major == 2:
+                    file_data = file_data.decode('utf-8').encode('utf-8')
+                else:
+                    file_data = file_data.encode('utf-8')
```

Comment:
I think this looks better. If in the future using a try-except is easier just add a comment that this is expected to fail on Python2/3 and not for some other reason.